### ISSUE TITLE
Usage documentation on radio button wrongly uses 'checkbox' intermittently.

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#9483]
in radio buttons [Usage](https://www.carbondesignsystem.com/components/radio-button/usage/), it states: “Users can trigger a state change by pressing Space while the checkbox input has focus.” First, there are a bunch of uses of “checkbox” used on the Usage tabpanel where it should say “radio button”. Looks like an oversight from copying content between the two similar pieces of guidance. For radio buttons, this would only ever be true if none of the radio buttons in a group are selected by default. Elsewhere you say “The default view of a radio button is having at least one radio button preselected.” So at the least, the first quote should be “In the unusual situation where none of the radio buttons in a group are selected by default, users can trigger a state change of the unselected radio button with focus by pressing Space .” Or, conversely, you could just take the whole sentence out. (Probably the easiest choice.)
